src/nativestore/PropertyLink.cpp

```diff
@@ -138,21 +157,25 @@ unsigned int PropertyLink::insert(std::string name, char* value) {
         pthread_mutex_unlock(&lockInsertPropertyLink);
         return this->blockAddress;
     }
+
+    // TODO(thevindu-w): Temporarily commented to resolve conflict
```

Comment:
Are there any side effects of this commenting?

Reply:
No side effects. Now it has the old code before optimization.

Reply:
Ok